### PR TITLE
Added page for 403 HTTP response

### PR DESCRIPTION
**Pull request by Newman101**
Added custom page for 403 HTTP response.

### CODE REVIEW

File: public/403.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+</head>
+
+<body>
+  <!-- This file lives in public/404.html -->
+  <div class="dialog">
+    <h1>Access denied.</h1>
+    <p>You do not have a permission to view this page.</p>
+  </div>
+</body>
+</html>
```
Comment (jamiecobbett) [inline public/403.html]: Valid Unix files have a trailing newline :leftwards_arrow_with_hook:

File: public/403.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      border: 1px solid #ccc;
+      border-right-color: #999;
+      border-bottom-color: #999;
+    }
+    h1 { font-size: 100%; color: #f00; line-height: 1.5em; }
+  </style>
+</head>
+
+<body>
+  <!-- This file lives in public/404.html -->
```
Comment (bradleywright) [inline public/403.html]: This comment is wrong.

**Comment by evilstreak** [comment]
Thanks for the PR @Newman101.

Due to way our infrastructure is configured, ours users won't actually see this page. What's your use case for this change? Merging this change isn't going to cause us any problems, so if it's useful for some other users of the codebase we'd be happy to do so.

**Comment by Newman101** [comment]
Hi all. My use case for this change was simple, I noticed that you did not have 403 so I decided to add it, just in case it might be useful at some point. I'll just fix the comment mentioned above.

**Comment by bradleywright** [comment]
Thanks for this @Newman101, change looks good now.

**Pull request merged**